doc-content/enterprise-only/installation/supported-repos-con.adoc

```diff
@@ -0,0 +1,22 @@
+[id='supported-repos-con_{context}']
+
+= {PRODUCT} supported repositories
+You can use Git repositories and Apache Maven with {PRODUCT}.
+
+== Git repositories
+Git repositories are used internally within {CENTRAL} to store all processes, rules, and other artifacts that are created in the authoring environment. Git is a distributed version control system. It implements revisions as commit objects. When you commit your changes into a repository, a new commit object in the Git repository is created. When you create a project in {CENTRAL} it is added to the Git repository connected to {CENTRAL}.
+
+If you have projects in other Git repositories, you can import them into {CENTRAL} spaces. You can use Git hooks to synchronize the internal Git repositories with external repositories.
+
+== Apache Maven
+http://maven.apache.org/[Apache Maven] is a distributed build automation tool used in Java application development to build and manage software projects. You can use Maven to build, publish, and deploy your {PRODUCT} projects. Maven provides the following benefits:
+
+* The build process is easy and a uniform build system is implemented across projects.
+* All of the required JAR files for a project are made available at compile time.
+* A proper project structure is configured.
+* Dependencies and versions are well managed.
```

Comment:
"well managed" Should this be hyphenated? I don't believe so in this context, but want to check.

Reply:
I don't think so. If it were an adverbial phrase, then yes. As in, Michele is a well-managed reviewer.